Reporting/M5 Document/Reflection.md

```diff
@@ -0,0 +1,29 @@
+## How did your project management work for the team?  What was the hardest thing, and what would you do the same/differently the next time you plan to complete a project like this? 
```

Comment:
Add the following:
- Our use of a project Kanban proved to be highly effective as it enabled everyone to be aware of what everyone else is working on across the whole system
- Project management suffered significantly as the project scaled and more complex began to be integrated (direct messaging, text channels and friend requests), where we observed a significant reduction in team coordination. This was due to everything becoming interconnected and only one person developing the messaging system, which is the majority of the project's core focus. In the future, we would need to ensure that everyone understands how every part of the system works such that major tasks like messaging can be broken down across multiple team members.